index.html

```diff
@@ -159,8 +159,61 @@ <h3 class="u-color-grey-dark">Social Media</h3>
     </div>
     <!-- Social Media container -->
 
-       <!-- Testimonials -->
-       <section class="c-section">
+
+    <!-- Pricing section -->
+    <section class="c-section u-bkg-grey-light">
+        <div class="u-container">
+            <h2 class="c-section__title u-text-center">pricing options</h2>
+            <hr class="u-line u-bkg-warning mx-0">
+            <p class="u-text-center u-mt">Lorem ipsum dolor sit amet,consectetur adipiscing elit.</p>
+            <div class="c-grid c-grid--three c-grid--gap u-align-center u-mt u-mx">
+                <div class="c-pricing__card u-bkg-white u-mt">
```

Comment:
`c-pricing__card` bi trebalo da bude `c-pricing-card` jer je to blok, a ne element nekog drugog bloka.